src/librustc_trans/trans/mir/operand.rs

```diff
@@ -94,6 +94,24 @@ impl<'bcx, 'tcx> MirContext<'bcx, 'tcx> {
 
         match *operand {
             mir::Operand::Consume(ref lvalue) => {
+                // watch out for temporaries that do not have an
```

Comment:
Oh right! I wanted to preserve the memcpy for lvalue-to-lvalue copies, but for immediate values, the load/store combo is even actually better I suppose.